src/nameutil.rs

```diff
@@ -42,6 +42,20 @@ pub fn crate_name(name: &str) -> String {
     }
 }
 
+pub fn exported_crate_name(crate_name: &str) -> String {
```

Comment:
Maybe add some docs to these functions here to explain what they are actually returning and in which context that returned string is used

Reply:
Updated